app/Group.php

```diff
@@ -243,6 +244,27 @@ public function getLocation()
         return rtrim($this->location);
     }
 
+    public function canDelete() {
```

Comment:
Minor one - not for now but for retro - PHP code guidelines that I've seen (PSR-2, Laravel's own) tend to go with opening brace on new line.

Reply:
I've changed my IDE.